src/components/Item/Item.component.jsx

```diff
@@ -34,14 +55,52 @@ const Item = props => {
             firebase.dataBase
               .collection(localToken)
               .doc(documentId)
-              .update({ lastPurchaseDate: date });
+              .update({
+                lastPurchaseDate: date,
+                numberOfPurchases: numberOfPurchases,
+                nextPurchaseInterval: nextPurchaseInterval,
+                latestInterval: latestInterval,
+                lastEstimate: lastEstimate,
+              });
           }
         });
       });
   };
 
+  useEffect(() => {
+    if (over24 === false) {
```

Comment:
I think this would be best placed into its own function. 

When I first look at this use effect, I don't really know what it's supposed to do. If you moved it into a function possibly named `applyResupplyItemStyles` it would be more clear as to what it does 😄